packages/cli/src/InternalHooks.ts

```diff
@@ -488,7 +492,7 @@ export class InternalHooks {
 				? this.eventBus.sendWorkflowEvent({
 						eventName: 'n8n.workflow.success',
 						payload: sharedEventPayload,
-					})
+				  })
```

Comment:
Some formatting issues to autofix.

Reply:
prettier got got confused here, I did refactor code slightly